terms/PsychENCODESpecific/grant.json

```
@@ -0,0 +1,124 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id":"sysbio.PECSpecific-annotations.grant-0.0.1",
+    "description": "Grant number the data is being submitted for",
```

I would use the same description for the general purpose grant.
```suggestion
    "description": "Grant number including activity code, institute code, and serial number (e.g. U01MH103392)",
```